tests/validation/tests/v3_api/test_encryption_secrets.py

```diff
@@ -0,0 +1,48 @@
+from .common import *  # NOQA
+import pytest
+from .test_rke_cluster_provisioning import rke_config
+
+providers = [{"aescbc": {"keys": [{"name": "k-fw5hn", "secret": "RTczRjFDODMwQzAyMDVBREU4NDJBMUZFNDhCNzM5N0I="}]}}, {"identity": {}}]
+custom_resources = [{"resources": ["secrets"], "providers": providers}]
+custom_config = {"apiVersion": "apiserver.config.k8s.io/v1", "kind": "EncryptionConfiguration", "resources": custom_resources}
+SECRET_ENCRYPTION_CONFIG = {"customConfig": custom_config, "enabled": True, "type": "/v3/schemas/secretsEncryptionConfig"}
+
+def test_create_custom_cluster_with_encryption():
```

Comment:
Can you write some code to clean up this cluster when done with the test ?

Reply:
Addressed in changes.